src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String CSV_SEPARATOR = ",";
+    public static final int OPERATION_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+    public static final String BUY_NANE = "buy";
+    public static final String SUPPLY_NANE = "supply";
+    public static final String RESULT_NANE = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeStatisticToFile(readStatisticFromFile(fromFileName), toFileName);
+    }
+
+    private Statistic readStatisticFromFile(String fromFileName) {
+        Statistic statistic = new Statistic();
+        try (BufferedReader BufferedReader = new BufferedReader(
+                        new FileReader(new File(fromFileName))
+        )) {
+            String record = BufferedReader.readLine();
+
+            while (record != null) {
+                String[] splitRecord = record.split(CSV_SEPARATOR);
+                if (splitRecord[OPERATION_INDEX].equals(SUPPLY_NANE)) {
+                    statistic.addSupply(Integer.parseInt(splitRecord[VALUE_INDEX]));
+                }
+                if (splitRecord[OPERATION_INDEX].equals(BUY_NANE)) {
+                    statistic.addBuy(Integer.parseInt(splitRecord[VALUE_INDEX]));
+                }
+                record = BufferedReader.readLine();
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("File for reading: " + fromFileName + "does not found.", e);
+        } catch (IOException e) {
+            throw new RuntimeException("IOException when work with " + fromFileName, e);
```

Comment:
lets catch just IOException
FileNotFoundException extends IOException
different catch blocks are usually used when completely different exceptions are thrown, or you have different cases for similar exceptions